ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/AwsSecurityHubIntegrationForm.tsx

```diff
@@ -208,15 +209,19 @@ function AwsSecurityHubIntegrationForm({
                         touched={touched}
                         errors={errors}
                     >
-                        <FormSelect
+                        <SelectSingle
                             id="notifier.awsSecurityHub.region"
                             value={values.notifier.awsSecurityHub.region}
-                            onChange={(event, value) => onChange(value, event)}
-                            onBlur={handleBlur}
```

Comment:
Integration test fails because `onBlur` prop deleted. It looks like `SelectSIngle` supports it.

However, it might fail later in the test if `select` method is for form select.
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/cypress/integration/integrations/notifiers.test.js#L80

Here is an alternative to consider:
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/cypress/integration/integrations/machineAccessConfigs.test.ts#L37-L38